manual/sphinx/user_docs/advanced_install.rst

```diff
@@ -213,37 +213,33 @@ The following configuration commands have been used successfully:
     make
     make install
 
-* netCDF-4.7.4::
+* netCDF-4.7.4 (note: using cmake to build gave errors in ``make test`` while the autotools build with ``configure`` passed all tests in ``make check``; netCDF-4.8.0 failed to link to hdf5-1.12.0 and failed tests with both cmake and autotools)::
 
-    mkdir build
-    cd build
-    cmake -DCMAKE_INSTALL_PREFIX=/directory/to/install/netcdf -DCMAKE_BUILD_TYPE=Release ..
+    CPPFLAGS="-I<prefix for hdf5>/include" LDFLAGS="-L<prefix for hdf5>/lib/" ./configure --prefix=/directory/to/install/netcdf
     make
     make install
 
-* netCDF-cxx4-4.3.1::
+* netCDF-cxx4-4.3.1 (note: cmake build works, but installs to a ``lib64`` subdirectory, while autotools installs to ``lib``, so easier to use autotools to match netCDF)::
 
-    mkdir build
-    cd build
-    cmake -DCMAKE_INSTALL_PREFIX=/directory/to/install/netcdf -DCMAKE_BUILD_TYPE=Release ..
+    CPPFLAGS="-I<prefix for hdf5>/include -I<prefix for netcdf>/include" LDFLAGS="-L<prefix for hdf5>/lib/ -L<prefix for netcdf>/lib/" ./configure --prefix=<prefix for netcdf>
```

Comment:
rather then repetetive `<prefix ...>` it would be nice to set `NETCDFDIR=<prefix ...>` and then use `$NETCDFDIR`